qrhl/qrhl.el

```diff
@@ -0,0 +1,115 @@
+;; Author: Dominique Unruh
```

Comment:
Please add a complete header with license claim etc., at the beginning of your file, for instance:

```suggestion
;;; qrhl.el --- Mode for qrhl-tool theorem prover -*- coding: utf-8; lexical-binding: t -*-

;; This file is part of Proof General.

;; Copyright © YYYY-TODO  Dominique Unruh

;; Author: Dominique Unruh

;; SPDX-License-Identifier: GPL-3.0-or-later

;;; Commentary:
;;
;; See also https://dominique-unruh.github.io/qrhl-tool/
;;

;;; Code:
```

Reply:
Added license (and removed coding: utf-8 headers) in afb17bb